styles/main.css

```
@@ -0,0 +1,244 @@
+  background-color: var(--color-lightblue-90);
+  border-radius: 8px;
+  color: var(--color-white);
+  font-weight: 400;
+  font-size: 1rem;
+}
+
+.see-more-button {
+  width: 100%;
+  height: 3.0625rem;
+
+  color: var(--color-white);
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)